boards/common/arduino-mkr/Makefile.dep

```diff
@@ -1,3 +1,8 @@
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
+
+FEATURES_REQUIRED += bootloader_arduino
```

Comment:
Guard this with an environment variable?

Reply:
Or honestly, just dont add it, just leave it as something the user can add.

Reply:
By default, I want this feature to used for these boards.